CMakeLists.txt

```diff
@@ -549,8 +549,12 @@ else()
     set(_VS_PLATFORM_TOOLSET_VERSION "${_VS_PLATFORM_TOOLSET_VERSION}.0")
     set(BOOST_CONFIGURE_FLAGS
         debug-store=database
-        address-model=32
     )
+    if(CMAKE_CL_64)
```

Comment:
Discouraged by CMake upstream, you should use `CMAKE_SIZEOF_VOID_P` instead. But that's no big problem.

Reply:
I could change it.

Reply:
Fixed.